.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -177,7 +266,7 @@ demisto-sdk-nightly:test-upload-flow:
   script:
     - !reference [test-upload-flow, script]
 
-demisto-sdk-nightly:run-commands-against-instance:
+demisto-sdk-nightly:run-end-to-end-tests-general:
```

Comment:
what's "general" aren't you testing against an instance?

Reply:
Those are tests that are not being tested against an instanse